android/src/main/java/com/neuroidreactnativesdk/NeuroidReactnativeSdkModule.kt

```diff
@@ -38,54 +32,47 @@ class NeuroidReactnativeSdkModule(reactContext: ReactApplicationContext) :
             val neuroID = NeuroID.Builder(application, key).build()
             NeuroID.setNeuroIdInstance(neuroID)
         }
-        NeuroID.getInstance()?.configureWithOptions(key, null)
 
         val reactCurrentActivity = currentActivity
         if (reactCurrentActivity != null) {
             NeuroID.getInstance()?.setForceStart(reactCurrentActivity)
```

Comment:
setForceStart has been deleted in the android sdk. Should this be changed registerPageTargets instead?

Reply:
good catch, updated